### DISCUSSION TITLE
creating Amharic model am_core_web_sm

### DISCUSSION THREAD

**Discussion by yosiasz** [explosion/spaCy#6584] [Language Support]
Greetings,

How do I go about create a small Amharic model?

Thanks

**Comment by yosiasz** [comment]
found a template. 3 years old but might work

https://github.com/explosion/spacy-models/tree/master/template

**Comment by ines** [comment]
The template you linked is just what's used internally by spaCy to generate an installable Python package for a trained model.

This thread on adding models for new languages should be a good place to get started: #3056 If you have suitable training data for your model, the first step would be to convert the data for use with spaCy. Also see the documentation on training via the CLI: https://spacy.io/usage/training#spacy-train-cli

**Reply by yosiasz** [reply]
@ines thanks, looks like there is one here https://github.com/UniversalDependencies/UD_Amharic-ATT

**Reply by yosiasz** [reply]
@ines few year late but back to the party ;) can the above from UD be used or do I need to create one of my own? guidance would be greatly appreciated!

**Reply by JephteyAdolphe** [reply]
update?
